phpBB/styles/prosilver/theme/content.css

```diff
@@ -517,10 +516,9 @@ blockquote .codebox {
 	display: block;
 	height: auto;
 	max-height: 200px;
-	padding-top: 5px;
+	padding: 8px 3px 5px 3px;
```

Comment:
```
<div class="codebox">
<p>...</p>
<pre><code></code></pre>
</div>
```

The codebox initially had 3px padding for all sides, but it was unwanted since it created margins for the scrollbars and so I've removed it.

So the `<code>` then needed 3px padding that was lost. However, I removed 2px bottom and top margin from the `<code>` for the sake of using just padding, so I added those 2px as bottom and top padding -> `padding: 5px 3px 5px 3px;`

Plus 3px top padding (I removed 3px bottom margin from `<p>`) -> `padding: 8px 3px 5px 3px;`